CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+## [2.0](https://github.com/react-native-community/react-native-viewpager/compare/master...2.0)
```

Comment:
Can you remove this `CHANGELOG` file? We will add this in to the Releases section.